serverless-configs/aws-infrastructure-resources.yml

```diff
@@ -138,6 +138,53 @@ Resources:
                   - states:*
                 Resource: '*'
 
+  # Redis Cache for browse-scaler
+  RedisSecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: Ingress for Redis Cluster
+      VpcId: ${env:VPC_ID}
+      SecurityGroupIngress:
+      - IpProtocol: tcp
+        FromPort: 1521
+        ToPort: 1521
+
+  RedisCacheSubnetGroup:
+    Type: AWS::ElastiCache::SubnetGroup
+    Properties:
+      CacheSubnetGroupName: browse-scaler-${self:provider.stage}
+      Description: 'Redis Cache Subnet Group'
+      SubnetIds:
+        - ${env:SUBNET_ID_A}
+        - ${env:SUBNET_ID_B}
+
+  RedisParameterGroup:
+    Type: AWS::ElastiCache::ParameterGroup
+    Properties:
+      CacheParameterGroupFamily: redis7
+      Description: 'Redis ElasticCache Parameter Group'
+      Properties:
+        maxmemory-policy: allkeys-lru
+
+  RedisElasticCacheCluster:
+    DependsOn: RedisSecurityGroup
+    Type: AWS::ElastiCache::CacheCluster
+    Properties:
+      Engine: redis
+      EngineVersion: '7.0'
+      Port: 1521
```

Comment:
The default port seems to be `6379 ` I only bring this up because trying to work on this locally is causing some issues with it being difficult to change this port for a running `redis` cache in a docker container rather than using the base image during local dev where you need the cache up.